code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -120,7 +120,7 @@
 			new /obj/item/card/emag/doorjack(src) // 3 tc
 
 /obj/item/storage/box/syndicate/bundle_b/PopulateContents()
-	switch (pickweight(list( "bond" = 2, "ninja" = 1, "darklord" = 1, "white_whale_holy_grail" = 2, "mad_scientist" = 2, "bee" = 1, "mr_freeze" = 2)))
+	switch (pickweight(list( "bond" = 2, "ninja" = 1, "darklord" = 1, "white_whale_holy_grail" = 2, "mad_scientist" = 2, "bee" = 1, "mr_freeze" = 2, "ancient" = 1)))
```

Comment:
Split the selection out into its own line, make the weights across a multi-line list, and make these local file defines.

Reply:
> defines

That's going to take me at least a couple hours to figure out how to write defines, at minimum...